src/org/funda/prog101/enrique/udacityTraining/src/Chess/Bishop.java

```diff
@@ -0,0 +1,18 @@
+package Chess;
+
+public class Bishop extends Piece{
+	
+	public Bishop(int row, int column) {
+		super(row, column);
+		// TODO Auto-generated constructor stub
+	}
+
+	public boolean isValidMove(Position newPosition) {
+		if(Math.abs(newPosition.row - this.position.column) == Math.abs(newPosition.row - this.position.row)) {
+			return true;
+		}else {
+			return false;
+		}
```

Comment:
you can just return: Math.abs(newPosition.row - this.position.column) == Math.abs(newPosition.row - this.position.row)